Directory.Packages.props

```diff
@@ -101,6 +101,7 @@
     <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.2.0" />
     <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.7.2" />
     <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components.Icons" Version="4.7.2" />
+    <PackageVersion Include="Milvus.Client" Version="2.3.0-preview.1"/>
```

Comment:
I see there isn't a stable version of this yet. Do we know when it plans on going stable?

For now, we will need to add 

https://github.com/dotnet/aspire/blob/756dfe663d0759aa0d2a0e1208502a403a398c32/src/Components/Aspire.Azure.AI.OpenAI/Aspire.Azure.AI.OpenAI.csproj#L6-L7

to the component .csproj since we can't ship stable until all dependencies are stable.